FogCreek/Glitch-Community
# Hilarious pumpkin 1: componentize project profile and add avatar upload

## Links
* Remix link: https://hilarious-pumpkin-1.glitch.me/
* Manuscript link: https://glitch.manuscript.com/f/cases/3328422/

## GIF/Screenshots:
![Screen Shot 2019-04-25 at 3 36 54 PM](https://user-images.githubusercontent.com/938722/56763322-1e11ec80-6770-11e9-9faf-3066b25b55b6.png)
![Screen Shot 2019-04-25 at 3 36 42 PM](https://user-images.githubusercontent.com/938722/56763327-20744680-6770-11e9-8ec1-a6e7cbf00bfb.png)

## Changes:
* use ProfileContainer on project page
* add avatar upload to ProjectEditor

## How To Test:
* visit https://hilarious-pumpkin-1.glitch.me/ while logged in
* verify that project profiles you do not own render correctly, e.g. https://hilarious-pumpkin-1.glitch.me/~nails
* create a new project and verify that its profile renders correctly (with editable fields)
* upload an avatar to your project
